### ISSUE TITLE
multiple commands (NVDA+Shift+2, NVDA+Shift+5) report same information (used and total space for both physical and virtual RAM)

### ISSUE THREAD

**Issue by rpccl** [josephsl/resourceMonitor#59]
As of now, both the commands gives same information. This is not an ishue as commands are wel documented. But this question comes from discussion about audit of add-ons in the store. I strongly felt that there should be a strong reason as to why both commands exist.  To find it, I looked into closed ishues and pool requests. I found nothing. Perhaps git commits might reveal something, but I am not that proficient with it. I am creating an ishue, so that this becomes publick info and it willl be easyer for anyone to figure out by themselves.
If there is no strong reason, 1 of the command should be removed.
And there is a strong possibility that I am missing something vary obvious. If so, correct me.
Thanks.

**Comment by josephsl** [comment]
Hi,

Reason: NVDA+Shift+number row 2 keyboard combination does not work on some keyboards (I have used a couple keyboards exhibiting this problem), so NVDA+Shift+number row 5 is the alternative key combination.

Thanks.

**Comment by rpccl** [comment]
Thanks,
Then, ithor the key combo (NVDA+Shift+Number row 2) can be removed or safest option is to include the reasone in the documentation.
Thanks.

**Comment by josephsl** [comment]
Hi,

The second option would be better in this case (a note about duplicate key assignments). If done, it will be part of Resource Monitor 26.08.

Thanks.

**Comment by rpccl** [comment]
I agree. Thanks.

**Comment by josephsl** [comment]
Resolved.
